src/QsCompiler/Compiler/RewriteSteps/Monomorphization.cs

```diff
@@ -28,16 +30,21 @@ internal class Monomorphization : IRewriteStep
 
         public bool ImplementsPostconditionVerification => true;
 
-        public Monomorphization()
+        /// <summary>
+        /// Constructor for the Monomorphization Rewrite Step. If keepAllIntrinsics is
+        /// set to true, intrinsics will not be removed as part of the rewrite step.
```

Comment:
It would be good to move the description of the `keepAllIntrinsics` parameter to a `<param>` tag.

Reply:
done